dependencies.props

```diff
@@ -158,7 +158,7 @@
     </StaticDependency>
 
     <StaticDependency Include="Microsoft.TargetingPack.Private.WinRT">
-      <Version>1.0.3</Version>
+      <Version>1.0.5</Version>
```

Comment:
I thought you were going to remove this. It's not used so we should just get rid of it.